config/features.yml

```diff
@@ -8,3 +8,6 @@ features:
     description: >
       On https://www.va.gov/find-locations/ enable veterans to search for Community care by showing that option in the "Search for" box.
       This toggle is owned by Rian
+  profile_show_direct_deposit:
```

Comment:
perfect thank you!

Reply:
One thing I'm wondering about now is if we want to have a singular toggle for "enable the API endpoint for direct deposit" and "show it in the UI".  If we had reason to shut it off we might want to make sure the backend also won't accept requests to update the banking info.

Reply:
I think separate toggles work for short lived toggles (release toggles) but and that long term toggles (ops toggle) would cover it both systems. 

We also have the option to poll the toggle values and live update ui.